analyses/comparative-RNASeq-analysis/run-comparative-RNAseq.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# CCDL for ALSF 2021
+# Joshua Shapiro
+#
+# Set so the script stops if there is an error
+set -euo pipefail
+
+# Set the working directory to the directory of this file
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
```

Comment:
Very minor comment: might want to use a variable for `../../data` here.

Reply:
done!